2017/SEPT.md

```diff
@@ -26,7 +26,7 @@
 | Contest | Problem | [ECR](#ecr) | C | C++ | Java | Python |
 |:--------------|:----------------:|:----------------:|:----------------:|:----------------:|:-----------------:|:-----------------:|
 | [COOK86](https://www.codechef.com/COOK86) | [LIKECS01](https://www.codechef.com/COOK86/problems/LIKECS01) | ★★ | | | | |
-| [COOK86](https://www.codechef.com/COOK86) | [LIKECS02](https://www.codechef.com/COOK86/problems/LIKECS02) | ★★★ | | | | |
+| [COOK86](https://www.codechef.com/COOK86) | [LIKECS02](https://www.codechef.com/COOK86/problems/LIKECS02) | ★★★ | |  [![image](../img/CC.png)](https://www.codechef.com/viewsolution/15449918)  [![image](../img/GH.png)](SEPT/COOK86/LIKECS02/LIKECS02.cpp) (100 pts)  [![image](../img/AC.png)](#) | | |
```

Comment:
Why extra space after `|` ? Please be consistent with spacing all over the README.

Reply:
Oh shit, why did he gave 2 spaces there? Please fix it.